src/Swarm/Language/Typecheck.hs

```diff
@@ -476,6 +477,7 @@ inferConst c = toU $ case c of
   Exp -> arithBinT
   AppF -> [tyQ| (a -> b) -> a -> b |]
   As -> [tyQ| string -> {cmd a} -> cmd a |]
+  Poll -> [tyQ| cmd a |] -- TODO: poll is NOT exported as command
```

Comment:
Right, `Poll` does not have any sensible type---yet another reason it should be a `Frame` instead of a `Const`.